### DISCUSSION TITLE
When is your next release?

### DISCUSSION THREAD

**Discussion by quocviet101** [libgd/libgd#753] [Q&A] [answered]
Hello,
We are using libgd in our project and security team scanned out this vulnerability https://nvd.nist.gov/vuln/detail/CVE-2021-40145. I understand that the fix is merged to master. When do you release 2.3.3 to fix that?
Thank you very much.

**Comment by pierrejoye** [comment, marked as answer]
Hi @quocviet101 

Tomorrow, 2.3.3 3PM BKK time , you can see the fixed in the issues by selecting the 2.3.3 milestone.

2.4.0 will be the main focus afterwards, together with 3.0 (see the wiki page [3.0 Roadmap](https://github.com/libgd/libgd/wiki/3.0-Roadmap)

**Reply by quocviet101** [reply]
Thank you very much for the information. Best.
